espaloma/mm/nonbonded.py

```diff
@@ -7,17 +7,17 @@
 # CONSTANTS
 # =============================================================================
 import espaloma as esp
+from simtk import unit
 
 # CODATA 2018
 # ref https://en.wikipedia.org/wiki/Coulomb_constant
-# K_E = (
-#     8.9875517923 * 1e9
-#     * unit.kilogram
-#     * (unit.meter ** 3)
-#     * (unit.second ** (-4))
-#     * (unit.angstrom ** (-2))
-#     ).value_in_unit(esp.units.COULOMB_CONSTANT_UNIT)
-
+K_E = (
+    8.9875517923 * 1e9
```

Comment:
Is this a global constant? If so, can you add a comment describing what it is supposed to represent?